google/cloud/storage/client.h

```diff
@@ -533,6 +533,40 @@ class Client {
     return raw_client_->PatchObject(request).second;
   }
 
+  /**
+   * Compose existing objects into a new object in the same bucket.
+   *
+   * @param bucket_name the name of the bucket that contains the object.
+   * @param destination_object_name the composed object name.
+   * @param source_objects objects used to compose `destination_object_name`.
+   * @param destination_object_metadata the new metadata for the Object. Only
+   *     the writeable fields accepted by the `Objects: compose` API are used,
+   *     all other fields are ignored.
+   * @param options a list of optional query parameters and/or request headers.
+   *     Valid types for this operation include `ComposeObjectRequest`,
+   *      `EncryptionKey`, `Generation`, `IfGenerationMatch`,
+   *      `IfMetagenerationMatch`, `PredefinedAcl`,
+   *      `SourceEncryptionKey`, `UserProject`.
+   *
+   * @throw std::runtime_error if the operation fails.
+   *
+   * @par Example
```

Comment:
Consider changing this to

```C++
/**
...
 * @par Examples
 *
 * @snippet storage_object_samples.cc compose object
 *
 * @snippet storage_object_samples.cc compose object from encrypted objects
```

So both of your examples are visible in the Doxygen documentation.